libmscore/excerpt.cpp

```diff
@@ -1034,6 +1079,9 @@ QList<Excerpt*> Excerpt::createAllExcerpt(MasterScore *score)
             if (part->show()) {
                   Excerpt* e   = new Excerpt(score);
                   e->parts().append(part);
+                  e->tracks().insert(part->startTrack(), 0);
+                  e->tracks().insert(part->startTrack() + 1, 1);
+                  e->tracks().insert(part->startTrack() + 3, 2);
```

Comment:
No, this is a "custom setting" I just wanted to extract voice 1, 2 and 4 of every stave to a single excerpt to check if this is working, too.